UPGRADE.md

```diff
@@ -1,5 +1,13 @@
 # Upgrade to 2.12
 
+## Deprecate `Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper`
+
+Using a console helper to provide the ORM's console commands with one or
+multiple entity managers had been deprecated with 2.9 already. This leaves
+The `EntityManagerHelper` class with no purpose which is why it is now
+deprecated too. Applications that still rely on the `em` console helper, you
+can easily recreate that class in their own codebase.
```

Comment:
At the beginning of that sentence, it feels like you are talking to applications themselves, and at the end to the persons maintaining them.

Reply:
Yeah, I rephrased the setence while writing it. 🫣